lib/bloom/components/badge.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Bloom.Components.Badge do
+  use Phoenix.Component
+
+  @moduledoc """
+  Badge component
+  """
+
+  attr(:variant, :string, doc: "Badge variant", default: "primary")
+
+  attr(:class, :string, default: "", doc: "Class for badge")
+  attr(:rest, :global)
+
+  slot(:inner_block, required: true)
+
```

Comment:
This is great - but one usecase I always have for badges is for tags and those require lots of colour variations.

Can you think of a way we can make them easily customisable from their colours without defining a variation?
Maybe a variation called colour and another prop that sets the colour? Could be done well with pattern matching.